File: python/492.py
import math
class Solution:
    def constructRectangle(self, area):
        """
        :type area: int
        :rtype: List[int]
        """
        a = int(math.sqrt(area))
        if a * a == area:
            return [a,a]
        width = a
        while area % width != 0:
            width -=1
        return area//width,width

File: java/492.java
class Solution {
    public int[] constructRectangle(int area) {
        int i = (int)Math.pow(area,0.5);
        int[] ans = new int[2];
        while(i > 0){
            if(area % i == 0){
                ans[0] = area/i;
                ans[1] = i;
                return ans;
            }
            i--;
        }
        return ans;
    }
}